x-pack/plugins/apm/server/routes/alerts/register_transaction_duration_alert_type.ts

```diff
@@ -188,6 +192,24 @@ export function registerTransactionDurationAlertType({
           windowSize: ruleParams.windowSize,
           windowUnit: ruleParams.windowUnit,
         });
+
+        // The function args wrapped with arrays because this function is shared with the frontend and the vars there are parsed by parseTechnicalFields()
+        const relativeViewInAppUrl = getAlertUrlTransactionDuration(
+          [ruleParams.serviceName],
+          [
+            getEnvironmentEsField(ruleParams.environment)?.[
+              SERVICE_ENVIRONMENT
+            ],
+          ],
+          [ruleParams.transactionType]
+        );
+
+        const viewInAppUrl = basePath.publicBaseUrl
```

Comment:
Can we have something like `const viewInAppUrl = basePath.prepend(relativeViewInAppUrl);`?

Reply:
Oh just seen that it's using `publicBaseUrl`